src/components/ProjectVersion/index.jsx

```diff
@@ -1,19 +1,178 @@
+  };
+
+  const onChangeBeforeVersion = ev => {
+    const version = ev.target.value;
+
+    setBeforeVersion(version);
+  };
+
+  const onChangeAfterDate = ev => {
+    const date = ev.target.value;
+
+    setAfterDate(date);
+  };
+
+  const onChangeAfterVersion = ev => {
+    const version = ev.target.value;
+
+    setAfterVersion(version);
+  };
+
+  const dateFilter = () => {
+    if (beforeDate) {
+      return allDates.filter(
+        dateString => new Date(dateString) > new Date(beforeDate),
+      );
+    }
+
+    return false;
+  };
+
+  const updateVersions = (prevVersion, nextVersion) => {
+    usePageStatusStore.getState().setStatus({
+      beforeVersion: prevVersion,
+      afterVersion: nextVersion,
+    });
+  };
+
+  const onClickBackToProject = ev => {
+    ev.preventDefault();
+
+    usePageStatusStore.getState().clearProject();
+
+    navigate(-1);
+  };
+
+  const onClickSubmitButton = async ev => {
+    ev.preventDefault();
+    setIsLoaded(false);
+    const result = { beforeVersion, afterVersion };
+
+    updateVersions(result);
+
+    const pageList = await getPages(result);
+
+    usePageListStore.getState().setPages(pageList);
+
+    setIsLoaded(true);
+    navigate("/page");
+  };
+
+  const prevData = {
+    label: "이전 버전",
+    description: "지정한 버전 명이 없으면 시간으로 보여요!",
+    selects: [
+      {
+        id: "beforeDate",
+        onChange: onChangeBeforeDate,
+        options:
+          allDates &&
+          allDates.map(date => (
+            <option key={date} value={date}>
+              {date}
+            </option>
+          )),
+      },
+      {
+        id: "beforeVersion",
+        onChange: onChangeBeforeVersion,
+        options:
+          beforeDate &&
+          Object.entries(byDates[beforeDate]).map(([key, value]) => (
+            <option key={key} value={key}>
+              {convertLabel(value.label)}
+            </option>
+          )),
+      },
+    ],
+  };
+
+  const nextData = {
+    label: "이후 버전",
+    description: "지정한 버전 명이 없으면 시간으로 보여요!",
+    selects: [
+      {
+        id: "afterDate",
+        onChange: onChangeAfterDate,
+        options:
+          dateFilter() &&
+          dateFilter().map(date => (
+            <option key={date} value={date}>
+              {date}
+            </option>
+          )),
+      },
+      {
+        id: "afterVersion",
+        onChange: onChangeAfterVersion,
+        options:
+          afterDate &&
+          Object.entries(byDates[afterDate]).map(([key, value]) => (
+            <option key={key} value={key}>
+              {convertLabel(value.label)}
+            </option>
+          )),
+      },
+    ],
+  };
+
+  const contents = {
+    title: {
+      step: "02",
+      firstSentence: "비교할 해당 피그마 파일의",
+      secondSentence: "이전 / 최신 버전을 입력해 주세요",
+    },
+    buttons: [
+      { text: "이전", usingCase: "solid", handleClick: onClickBackToProject },
+      { text: "다음", usingCase: "solid", handleClick: onClickSubmitButton },
```

Comment:
P1: 이벤트 핸들러 함수라는 네이밍에 적합하게 작성해주시면 좋을 것 같습니다 :)

```suggestion
      { text: "이전", usingCase: "solid", handleClick: handleClickPrevButton },
      { text: "다음", usingCase: "solid", handleClick: handleSubmitButton },
```

Reply:
아래와 같은 형식으로 함수명 변경하였습니다.
`handle` + `이벤트 타입` + `엘리먼트 타입`
변경된 이벤트 함수명은 아래와 같습니다.

onClickBackToProject => handleClickPrevButton
onClickSubmitButton => handleSubmitNextButton